Kickstarter-iOS/Features/MessageBanner/Controller/MessageBannerViewController.swift

```diff
@@ -95,20 +102,26 @@ public final class MessageBannerViewController: UIViewController, NibLoading {
       }
   }
 
-  public func showBanner(with type: MessageBannerType, message: String) {
+  public func showBanner(with type: MessageBannerType, message: String, dissmissable: Bool = true) {
     self.bannerType = type
+    self.dissmissable = dissmissable
     self.viewModel.inputs.update(with: (type, message))
     self.viewModel.inputs.bannerViewWillShow(true)
   }
 
   private func showViewAndAnimate(_ isHidden: Bool) {
+    if !isHidden, !self.view.isHidden, !self.dissmissable { return }
+
     let duration = isHidden ? AnimationConstants.hideDuration : AnimationConstants.showDuration
 
     let hiddenConstant = self.view.frame.height + (self.view.superview?.safeAreaInsets.bottom ?? 0)
 
     if !isHidden {
       self.view.superview?.bringSubviewToFront(self.view)
-      self.view.superview?.isUserInteractionEnabled = false
+
+      if self.dissmissable {
+        self.view.superview?.isUserInteractionEnabled = false
```

Comment:
Just to double-check, nowhere else needs this protection, right? Just to make sure we don't end up disabling the whole UI 😆

Reply:
correct. Only when the banner is dismissible do we want to disable interaction.